### ISSUE TITLE
-a option for sls endpoint deploy is ambiguous

### ISSUE THREAD

**Issue by assaflavie** [serverless/serverless#503]
```
Deploys REST API endpoints

  -s, --stage
	Optional if only one stage is defined in project

  -r, --region
	Optional - Target one region to deploy to

  -e, --aliasEndpoint
	Optional - Point Endpoint(s) to a specific Lambda alias

  -a, --aliasRestApi
	Optional - Override the API Gateway "functionAlias" Stage Variable

  -d, --description
	Optional - Provide custom description string for API Gateway stage deployment description

  -a, --all
	Optional - Deploy all Functions
```

Is `-a` supposed to mean two things (is it interpreted by context?). In any case, a bit confusing.

**Comment by ac360** [comment]
@assaflavie Thanks for pointing this out.  We'll fix this today.  :+1:

**Comment by ac360** [comment]
Fixed in v0.1.3.  Thanks @assaflavie
